minesweeper.py

```diff
@@ -287,6 +287,84 @@ def quit_game(event=None):
     root.destroy()
 
 
+highscore_window = None
+
+
+def create_highscores_window(event=None):
+    global highscore_window
+    if highscore_window is not None and highscore_window.winfo_exists():
+        highscore_window.lift()
+        return
+    highscore_window = tkinter.Toplevel()
+    highscore_window.resizable(False, False)
+    highscore_window.title("High Scores")
+    frame = ttk.Frame(highscore_window)
+    frame.pack(fill="both", expand=True)
+
+    treeview = ttk.Treeview(frame)
+
+    # Define columns
+    treeview["columns"] = ("Mine Percentage", "Time per Square", "Total Time")
+
+    # Format columns
+    treeview.column("#0", width=0, stretch="NO")
+    treeview.column("Mine Percentage", anchor="w", width=120)
+    treeview.column("Time per Square", anchor="w", width=120)
+    treeview.column("Total Time", anchor="w", width=100)
+
+    # Create headings
+    treeview.heading("#0", text="", anchor="w")
+    treeview.heading("Mine Percentage", text="Mine Percentage", anchor="w")
+    treeview.heading("Time per Square", text="Avg. Time per Square", anchor="w")
+    treeview.heading("Total Time", text="Total Time", anchor="w")
+
+    def get_highscore_data(highscore_dict):
+        highscore_mine_percentage = round(
+            highscore_dict["mine_count"]
+            / (highscore_dict["width"] * highscore_dict["height"])
+            * 100
+        )
+        highscore_avg_time = highscore_dict["time"] / (
+            highscore_dict["width"] * highscore_dict["height"]
+        )
+        return (-highscore_mine_percentage, highscore_avg_time)
+
+    def treeview_mine_percentage(highscore_dict):
```

Comment:
This is a somewhat long function name. Why not `get_mine_percentage`?

Because this function is defined inside `create_highscores_window()`, it is not visible to code outside `create_highscores_window()`, and the name doesn't need to include `treeview` or `highscore` in it.